src/Firebase/Factory.php

```diff
@@ -245,6 +257,39 @@ public function withClock(object $clock): self
         return $factory;
     }
 
+    public function createAppCheck(): Contract\AppCheck
+    {
+        $projectId = $this->getProjectId();
+        $serviceAccount = $this->getServiceAccount();
+
+        if ($projectId === '') {
+            throw new RuntimeException('Unable to determine the Firebase Project ID');
+        }
+
+        if ($serviceAccount === null) {
+            throw new RuntimeException('Unable to use AppCheck without credentials');
+        }
+
+        $http = $this->createApiClient([
+            'base_uri' => 'https://firebaseappcheck.googleapis.com/v1/projects/'.$projectId,
```

Comment:
```suggestion
            'base_uri' => 'https://firebaseappcheck.googleapis.com/v1/projects/'.$projectId.'/',
```

The base URI must end with a slash. Otherwise, a

```php
$http->get('GET', 'app/id')
```

will request `https://firebaseappcheck.googleapis.com/v1/projects/app/id` (which it did for me when I debugged the PR)